src/lbaselib.js

```diff
@@ -75,10 +75,13 @@ const {
     to_jsstring,
     to_luastring
 } = require("./fengaricore.js");
+const {
+    isBrowser
+} = require("./isnode.js");
 
 let lua_writestring;
 let lua_writeline;
-if (typeof process === "undefined") {
+if (isBrowser) {
```

Comment:
This check is actually checking for the `process` module (it uses `process.stdout` if available in the other branch)